2021-10-20 18:16:50,711 INFO    MainThread:99311 [wandb_setup.py:_flush():69] setting env: {}
2021-10-20 18:16:50,711 INFO    MainThread:99311 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-20 18:16:50,711 INFO    MainThread:99311 [wandb_init.py:_log_setup():348] Logging user logs to /home/pholur/Situation_Modeling/wandb/run-20211020_181650-3l0t8qjt/logs/debug.log
2021-10-20 18:16:50,711 INFO    MainThread:99311 [wandb_init.py:_log_setup():349] Logging internal logs to /home/pholur/Situation_Modeling/wandb/run-20211020_181650-3l0t8qjt/logs/debug-internal.log
2021-10-20 18:16:50,711 INFO    MainThread:99311 [wandb_init.py:init():381] calling init triggers
2021-10-20 18:16:50,711 INFO    MainThread:99311 [wandb_init.py:init():388] wandb.init called with sweep_config: {}
config: {}
2021-10-20 18:16:50,712 INFO    MainThread:99311 [wandb_init.py:init():430] starting backend
2021-10-20 18:16:50,712 INFO    MainThread:99311 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-20 18:16:50,755 INFO    MainThread:99311 [backend.py:ensure_launched():135] starting backend process...
2021-10-20 18:16:50,801 INFO    MainThread:99311 [backend.py:ensure_launched():140] started backend process with pid: 100117
2021-10-20 18:16:50,803 INFO    MainThread:99311 [wandb_init.py:init():435] backend started and connected
2021-10-20 18:16:50,805 INFO    MainThread:99311 [wandb_init.py:init():494] updated telemetry
2021-10-20 18:16:50,806 INFO    MainThread:99311 [wandb_init.py:init():517] communicating current version
2021-10-20 18:16:51,658 INFO    MainThread:99311 [wandb_init.py:init():522] got version response upgrade_message: "wandb version 0.12.5 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-20 18:16:51,658 INFO    MainThread:99311 [wandb_init.py:init():530] communicating run to backend with 30 second timeout
2021-10-20 18:16:51,794 INFO    MainThread:99311 [wandb_init.py:init():557] starting run threads in backend
2021-10-20 18:16:56,802 INFO    MainThread:99311 [wandb_run.py:_console_start():1605] atexit reg
2021-10-20 18:16:56,802 INFO    MainThread:99311 [wandb_run.py:_redirect():1479] redirect: SettingsConsole.REDIRECT
2021-10-20 18:16:56,803 INFO    MainThread:99311 [wandb_run.py:_redirect():1484] Redirecting console.
2021-10-20 18:16:56,806 INFO    MainThread:99311 [wandb_run.py:_redirect():1540] Redirects installed.
2021-10-20 18:16:56,806 INFO    MainThread:99311 [wandb_init.py:init():582] run started, returning control to user process
2021-10-20 18:17:08,491 INFO    MainThread:99311 [wandb_run.py:_atexit_cleanup():1575] got exitcode: 0
2021-10-20 18:17:08,508 INFO    MainThread:99311 [wandb_run.py:_restore():1547] restore
2021-10-20 18:17:11,506 INFO    MainThread:99311 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 734
  total_bytes: 734
}

2021-10-20 18:17:11,943 INFO    MainThread:99311 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 734
  total_bytes: 734
}

2021-10-20 18:17:12,068 INFO    MainThread:99311 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 734
  total_bytes: 734
}

2021-10-20 18:17:12,824 INFO    MainThread:99311 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 734
  total_bytes: 13325
}

2021-10-20 18:17:12,927 INFO    MainThread:99311 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 734
  total_bytes: 13736
}

2021-10-20 18:17:13,030 INFO    MainThread:99311 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13736
  total_bytes: 13736
}

2021-10-20 18:17:13,132 INFO    MainThread:99311 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13736
  total_bytes: 13736
}

2021-10-20 18:17:13,235 INFO    MainThread:99311 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13736
  total_bytes: 13736
}

2021-10-20 18:17:13,338 INFO    MainThread:99311 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13736
  total_bytes: 13736
}

2021-10-20 18:17:13,612 INFO    MainThread:99311 [wandb_run.py:_wait_for_finish():1697] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13736
  total_bytes: 13736
}

2021-10-20 18:17:13,894 INFO    MainThread:99311 [wandb_run.py:_wait_for_finish():1697] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 13736
  total_bytes: 13736
}
local_info {
}

2021-10-20 18:17:15,201 INFO    MainThread:99311 [wandb_run.py:_show_files():1947] logging synced files
